testapp/__main__.py

```diff
@@ -0,0 +1,50 @@
+import sys
```

Comment:
I'll add this in a separate PR, but unit style tests would be pretty useful as well imo